docs/react-testing-library.md

```diff
@@ -0,0 +1,72 @@
+| [![testing-library](./icons/other/testing-library.png)](undefined) | [no-manual-cleanup](https://github.com/testing-library/eslint-plugin-testing-library/tree/main/docs/rules/no-manual-cleanup.md)<br />Disallow the use of `cleanup`                                                           |                                                                                                                                          |         |           |
+| [![testing-library](./icons/other/testing-library.png)](undefined) | [no-promise-in-fire-event](https://github.com/testing-library/eslint-plugin-testing-library/tree/main/docs/rules/no-promise-in-fire-event.md)<br />Disallow the use of promises passed to a `fireEvent` method               |                                                                                                                                          |         |           |
+| [![testing-library](./icons/other/testing-library.png)](undefined) | [no-unnecessary-act](https://github.com/testing-library/eslint-plugin-testing-library/tree/main/docs/rules/no-unnecessary-act.md)<br />Disallow wrapping Testing Library utils or empty callbacks in `act`                   |                                                                                                                                          |         |           |
+| [![testing-library](./icons/other/testing-library.png)](undefined) | [no-wait-for-side-effects](https://github.com/testing-library/eslint-plugin-testing-library/tree/main/docs/rules/no-wait-for-side-effects.md)<br />Disallow the use of side effects in `waitFor`                             |                                                                                                                                          |         |           |
+| [![testing-library](./icons/other/testing-library.png)](undefined) | [no-wait-for-snapshot](https://github.com/testing-library/eslint-plugin-testing-library/tree/main/docs/rules/no-wait-for-snapshot.md)<br />Ensures no snapshot is generated inside of a `waitFor` call                       |                                                                                                                                          |         |           |
+| [![testing-library](./icons/other/testing-library.png)](undefined) | [prefer-query-by-disappearance](https://github.com/testing-library/eslint-plugin-testing-library/tree/main/docs/rules/prefer-query-by-disappearance.md)<br />Suggest using `queryBy*` queries when waiting for disappearance |                                                                                                                                          |         |           |
```

Comment:
Maybe also a warning?

Reply:
While it is a bad practice which reduces the visibility of errors, it technically does not break anything, so sure.